lib/util/index.js

```diff
@@ -106,18 +106,24 @@ function flatMap(list, mapFn) {
 
 /**
  * Guess the browser.
- * @returns {?string} browser - "chrome", "firefox", "safari", or null
+ * @returns {?string} browser - "chrome", "firefox", "safari", "edge", or null
  */
 function guessBrowser() {
-  if (typeof webkitRTCPeerConnection !== 'undefined') {
+  if (typeof window === 'undefined' || !window.navigator) {
+    return null;
+  }
+  var navigator = window.navigator;
+
+  if (navigator.webkitGetUserMedia) {
     return 'chrome';
-  } else if (typeof mozRTCPeerConnection !== 'undefined') {
+  } else if (navigator.mozGetUserMedia) {
     return 'firefox';
   } else if (typeof RTCPeerConnection !== 'undefined') {
-    if (typeof navigator !== 'undefined' && navigator.userAgent.match(/AppleWebKit\/(\d+)\./)) {
+    if (navigator.userAgent.match(/Edge\/(\d+)\./)) {
```

Comment:
Can you please reintroduce the `typeof navigator` check(s) such that this is safe to run in (Node) environments where `navigator` is `undefined`?